### ISSUE TITLE
program doesn't send code to telegram

### ISSUE THREAD

**Issue by sdferwfsdfsdf** [dark-hub-git/telegram-scraper-tool#1]
After I put in my api id, hash and phone number, the program doesn't send any code id to my  telegram account in order to connect it. can you help? thanks!

**Comment by dark-hub-git** [comment]
Log in with code and you won’t encounter problem

**Comment by sdferwfsdfsdf** [comment]
[*] Logging in from new accounts...

Please enter your phone (or bot token):

**Comment by dark-hub-git** [comment]
Input the number in the (+) format

**Comment by sdferwfsdfsdf** [comment]
I have tried with +, without any prefix..with different accounts...I just don't get the code back. 

I have also tested the telegram account with different programs and it works there, so it must be a problem at the code.

**Comment by dark-hub-git** [comment]
No, many didn’t encounter problems. I’m sure it’s the number you used, is it a celled number ? or a virtual number?
